dashboard/src/features/container/routes.js

```diff
@@ -0,0 +1,20 @@
+import Container from './components/Container'
+
+import { getAuthenticationStatus } from 'features/authn/actions'
+import { fetchCoreInfo } from 'features/core/actions'
+import { fetchTestnetInfo } from 'features/testnet/actions'
+
+const CORE_POLLING_TIME = 2 * 1000
+const TESTNET_INFO_POLLING_TIME = 30 * 1000
+
+export default (store) => ({
+  path: '/',
+  component: Container,
+  onEnter: () => {
```

Comment:
Correct. After checking the configuration on launch, the state is updated correctly. The most important change is checking for 401s with an unauthenticated client on launch.